### ISSUE TITLE
Double tapping is counter-intuitive on Android

### ISSUE THREAD

**Issue by Lonami** [HassanHeydariNasab/defendo#10]
There aren't many tower defense like games on F-Droid, so thanks for that! However, double tapping on a mobile phone is much less common than **long clicking**, my suggested behaviour. Both could be kept. Furthermore, the tower completely moves from where it should be! This occurs after trying to double tap several towers.

As a side note, it's strange the game doesn't fill the screen completely.

**Comment by HassanHeydariNasab** [comment]
The game is published for Desktop platform too and Double click came from there :D OK, I'll implement "long tap".
However, I used Godot 2.1.3 myself for developing and releasing it, but I used Godot 2.1.2 for F-droid build. I think this bug (undesirable move to the last tap position) is because of it. I'll try 2.1.3 or maybe 2.1.4.

I've set screen height static and width dynamic. It's because of limited board of the game. for example, first position of enemy. on Desktop it looks good.
